INSTALL.rst

```diff
@@ -144,22 +150,28 @@ From source
     # or:
     $ git clone git@github.com:USER/message_ix.git
 
-7. Open a command prompt in the ``message_ix`` directory and type::
+7. (Conditional) If you cloned your fork, add the main repository as a remote git repository. This will allow keeping up to date with changes there and importing tags, which also needs to be done for the install tests to succeed::
```

Comment:
Very minor, but I try to start each new sentence on a new line:
```
7. (Conditional) If you cloned … repository.
   This will allow … to succeed::
```

Reply:
Thanks, done :)